lib/src/main/kotlin/com/swmansion/starknet/data/responses/Transaction.kt

```diff
@@ -1,15 +1,19 @@
 package com.swmansion.starknet.data.responses
```

Comment:
Why should move those types to data/Transactions if we want to use them for both responses and requests.